src/test/java/roomescape/unit/service/ReservationServiceTest.java

```diff
@@ -0,0 +1,150 @@
+package roomescape.unit.service;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.anyLong;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static roomescape.common.Constant.FIXED_CLOCK;
+import static roomescape.integration.fixture.ReservationDateFixture.예약날짜_오늘;
+
+import java.time.LocalTime;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+import roomescape.domain.reservation.Reservation;
+import roomescape.domain.theme.Theme;
+import roomescape.domain.theme.ThemeDescription;
+import roomescape.domain.theme.ThemeName;
+import roomescape.domain.theme.ThemeThumbnail;
+import roomescape.domain.time.ReservationTime;
+import roomescape.repository.ReservationRepository;
+import roomescape.repository.ReservationTimeRepository;
+import roomescape.repository.ThemeRepository;
+import roomescape.service.ReservationService;
+import roomescape.service.request.ReservationCreateRequest;
+import roomescape.service.response.ReservationResponse;
+
+public class ReservationServiceTest {
```

Comment:
```suggestion
class ReservationServiceTest {
```
junit5부터는 테스트 클래스가 public하지 않아도 됩니다. 인텔리제이에서 자동으로 생성해도 public을 제거하여 생성해줍니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods